docs/Getting_Involved/Testing.rst

```diff
@@ -9,12 +9,25 @@ Executing our Tests
 *coala* has a big test suite. It is meant to work on every platform on
 every PC. If you just execute our tests you are doing us a favor.
 
-To run tests, you first need to install some dependencies. This can be
-done by executing:
+To run tests, you will need to open a file located in the default
+coala directory
+
+If you have not already, you can start by cloneing a copy of the project
+to your local machine.
+
+Either fork the repo on Github https://github.com/coala-analyzer/coala,
+or clone it directly from coala
 
 ::
 
-    $ pip3 install -r test-requirements.txt
+    $ git clone https://github.com/coala-analyzer/coala.git
+
+Then navigate to the directory where coala is located
+
+Next you need to install some dependencies. This can be
+done by executing:
```

Comment:
``done by executing  the following command while in the root of the coala project`` may be more helpful